github_rules/github_team_modified.py

```diff
@@ -0,0 +1,34 @@
+def rule(event):
+    if not event.get("action").startswith("team"):
+        return False
+    return (
+        event.get("action") == "team.add_member"
+        or event.get("action") == "team.add_repository"
+        or event.get("action") == "team.change_parent_team"
+        or event.get("action") == "team.create"
+        or event.get("action") == "team.destroy"
+        or event.get("action") == "team.remove_member"
+        or event.get("action") == "team.remove_repository"
+    )
+
+
+def title(event):
+    action = event.get("action")
+    if action.endswith(".create"):
+        action = "created team"
+    elif action.endswith(".destroy"):
+        action = "deleted team"
```

Comment:
I don't think there is anything wrong with this approach, it is straightforward to read and effective. But an alternative for consideration (assuming `event.get("action")` always returns a string with a `.` character):

```python
ACTION_MAPPINGS = {
  "create": "created team",
  "destroy": "deleted team",
  <...>
}
def title(event):
  action_key = event.get("action").split(".")[1]
  action = ACTION_MAPPINGS.get(action_key, event.get("action"))
  return (...)
```

Reply:
I like this! However, since some of these require f strings, I think I'd have to make the values lambdas, and call the lambdas? Which makes this a little more complicated (at least to read). Or I could make the mappings dict a local variable

Reply:
Ah I see what you mean now, we can't define it as a global mapping since the f strings rely on the contents of the event, and the lambda setup would be global but hard to read. I think what you settled on here works well.